Core.UnitTests/App_Packages/Remotion.Development.Moq.Decoration.Sources.1.17.6.0/DecoratorTestHelper.cs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// See the NOTICE file distributed with this work for additional information
+// regarding copyright ownership.  rubicon licenses this file to you under 
+// the Apache License, Version 2.0 (the "License"); you may not use this 
+// file except in compliance with the License.  You may obtain a copy of the 
+// License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software 
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT 
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the 
+// License for the specific language governing permissions and limitations
+// under the License.
+// 
+using System;
+using System.Linq.Expressions;
+using NUnit.Framework;
+using Remotion.Utilities;
+using Moq;
+
+// ReSharper disable once CheckNamespace
+namespace Remotion.Development.Moq.UnitTesting
+{
+  /// <summary>
+  /// Provides functionality for testing decorator methods that do nothing else but forward to the equivalent methods on a decorated object.
+  /// </summary>
+  /// <typeparam name="TInterface">The type of the interface.</typeparam>
+  public partial class DecoratorTestHelper<TInterface> 
```

Comment:
That does not seem right. Those RhinoMocks came from NuGet packages, Here's the file in re-motion: https://github.com/re-motion/Trunk-SVN2Git/blob/develop/SharedSource/Development/RhinoMocks/UnitTesting/DecoratorTestHelper.cs